op-chain-ops/genesis/config.go

```diff
@@ -171,7 +172,12 @@ type DeployConfig struct {
 	// GasPriceOracleOverhead represents the initial value of the gas overhead in the GasPriceOracle predeploy.
 	GasPriceOracleOverhead uint64 `json:"gasPriceOracleOverhead"`
 	// GasPriceOracleScalar represents the initial value of the gas scalar in the GasPriceOracle predeploy.
+	// DEPRECATED after Ecotone.
 	GasPriceOracleScalar uint64 `json:"gasPriceOracleScalar"`
```

Comment:
could add this depreciation warning also to the overhead

And with
```suggestion
	// GasPriceOracleOverhead represents the initial value of the gas overhead in the GasPriceOracle predeploy.
	// Deprecated: Since Ecotone, this field is superseded by GasPriceOracleBaseFeeScalar and GasPriceOracleBlobBaseFeeScalar.
	GasPriceOracleOverhead uint64 `json:"gasPriceOracleOverhead"`
	// GasPriceOracleScalar represents the initial value of the gas scalar in the GasPriceOracle predeploy.
	// Deprecated: Since Ecotone, this field is superseded by GasPriceOracleBaseFeeScalar and GasPriceOracleBlobBaseFeeScalar.
	GasPriceOracleScalar uint64 `json:"gasPriceOracleScalar"`
```
it's [picked up by godoc](https://github.com/golang/go/wiki/Deprecated/90c35e6078ceebb3efdcaac2fb31d852c3d5e88d) as an depreciation warning.